ui/frontend/Output/Gist.tsx

```diff
@@ -57,13 +57,15 @@ const Links: React.SFC = () => {
   const gistUrl = useSelector((state: State) => state.output.gist.url);
   const permalink = useSelector(selectors.permalinkSelector);
   const urloUrl = useSelector(selectors.urloUrlSelector);
+  const markdownSnippet = useSelector(selectors.snippetSelector);
 
   return (
     <Fragment>
       <Copied href={permalink}>Permalink to the playground</Copied>
       <Copied href={gistUrl}>Direct link to the gist</Copied>
       <Copied href={codeUrl}>Embedded code in link</Copied>
       <NewWindow href={urloUrl}>Open a new thread in the Rust user forum</NewWindow>
+      <Copied href={markdownSnippet}>Markdown snippet with all the information</Copied>
```

Comment:
While using the copy button here works as intended in the ticket, clicking the actual `href` that is being passed in will simply take you to the current page with everything in the snippet pasted after the url. I would advise to either not make this a link at all and just have the copy button. Or use some form of popup where the markdown is displayed as copy-able data. This way the user can see what they are copying. 

Lastly simply stating `markdown with all the information` is rather vague. I would suggest adding a name that is a bit more descriptive of the actual action taking place.